test/packages/good_v2/manifest.yml

```diff
@@ -1,4 +1,4 @@
-format_version: 2.0.0
```

Comment:
Update version in good_v2 to allow adding the new field in `_dev/build/build.yml`